yaksh/evaluator_tests/test_scilab_evaluation.py

```diff
@@ -71,6 +73,8 @@ def test_infinite_loop(self):
         result = get_class.evaluate(**kwargs) 
         self.assertFalse(result.get("success"))
         self.assertEqual(result.get("error"), self.timeout_msg)
+        time.sleep(4)
+        self.assertEqual(1, subprocess.call(["pgrep", "scilab-cli-bin"]))
```

Comment:
The over all code looks good, but I am unsure about the test cases.
pgrep a process may lead to unexpected results. In case scilab-cli-bin is run by a different process thread then the test case will fail.